docs/chapters/architecture/GENIVI-Lifecycle-Management.rst

```diff
@@ -0,0 +1,122 @@
+|                       |                          | for base case scenarios   |
++-----------------------+--------------------------+---------------------------+
+| BSP Developer         | Adapts PELUX to new      | Wants to integrate the    |
+|                       | platforms, builds        | GENIVI Life cycle         |
+|                       | systems based on PELUX   | Management components     |
+|                       |                          | on new platforms          |
++-----------------------+--------------------------+---------------------------+
+
+Constraints
+-----------
+
+The Life cycle management system should:
+
+- work on a typical embedded Linux context
+- be open source software
+
+Context and Scope
+-----------------
+
+The purpose of this section is to put the GENIVI Life cycle Management system into
+a broader context, and show how this subsystem interacts with other parts of the
+larger PELUX system.
+
+Technical Context
+^^^^^^^^^^^^^^^^^
+
+The life cycle cluster is a set of components that work closely with each other to
+define and implement the functionality required in the life cycle of a typical
+IVI scope.
+
+The functional scope of Life cycle is illustrated in this diagram:
+
+.. image:: https://at.projects.genivi.org/wiki/download/attachments/11567160/LC-Scope.png?version=1&modificationDate=1455743777000&api=v2
+
+Building Block View
+-------------------
+
+.. image:: https://at.projects.genivi.org/wiki/download/attachments/11567160/LC-StructureDefScope.png?version=1&modificationDate=1455743897000&api=v2
+
+Node State Manager
+^^^^^^^^^^^^^^^^^^
+The Node State Manager (NSM) is the central functional component that gathers
+information on the current running state of the embedded system. The NSM
+component provides a common implementation framework for the system state machine.
+It collates information from multiple sources and uses the data to determine the
+current state(s).
+
+Based on the given data, the NSM notifies registered consumers
+(applications or other platform components) of relevant changes in system state.
+Node state information can also be requested on-demand via provided D-Bus interfaces.
+The node state manager also provides shutdown management including shutdown request
+notification to consumers.
+
+The node state management is the last/highest level of escalation on the node and will
+issue the reset instruction and supply control logic. It is notified of errors and other
+status signals from components that are responsible for monitoring system health.
+
+Internally, node state management is made up of a common generic component, Node State
+Manager (NSM), and a system-specific state machine (NSMC) that is plugged into the Node
+State Manager. Through this architecture there can be a standardized solution with stable
+interfaces towards the applications, which still allows for product-specific behavior
+through the definition of the specific state machine. [1]_
```

Comment:
I dont think this is how we are supposed to use references. [Here](https://pelux.io/software-factory/master/swf-blueprint/docs/articles/intro/how-to-use-the-swf-platform-blueprint.html) at the end you have a link as an example.

Reply:
but this is the same